src/components/FormikMultiSelect.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { InputLabel, Select, MenuItem } from '@material-ui/core';
+import PropTypes from 'prop-types';
+
+const MenuProps = {
+  getContentAnchorEl: null, //TODO: remove this when material-ui gets updated to version 5
+  anchorOrigin: {
+    vertical: 'bottom',
+    horizontal: 'left',
+  },
+  PaperProps: {
+    style: {
+      maxHeight: 250,
+      width: 250,
+    },
+  },
+};
+
+function FormikMultiSelect(props) {
+  const { label, id, labelStyle, formik, options } = props;
+
+  return (
+    <>
+      <InputLabel className={labelStyle}>{label}</InputLabel>
+      <Select
```

Comment:
Optional: Have you considered setting the `renderValue` so that chips are displayed? This'd help show all the selected options.

Alternatively you may want the caller to provide a custom `renderValue` instead. E.g. maybe for grade level, if all High school grades are selected we display "All high schoolers" instead of "HS Freshman, HS Sophomore, ....". 

Again, this is optional so maybe leave for a later PR.

Reply:
yeah for sure I can explore this in another PR. The only thing with chips though, is that they might enlarge the Select field and that might look a little weird.

Reply:
Yeah enlarging the Select field may look a bit weird. But maybe that's ok. On the other hand, maybe in some cases displaying a [limited number of chips](https://mui.com/components/autocomplete/#limit-tags) makes more sense. Maybe a custom `renderValue` prop would be good.